assets/css/styleguide.scss

```diff
@@ -308,8 +304,15 @@ a#skipnav {
   }
 
   p, a {
```

Comment:
No I'm saying to add a note to see why it wouldn't be in the main pattern library css.